examples/survival_analysis/survival_analysis.ipynb

```diff
@@ -28,13 +28,15 @@
    "metadata": {},
```

Comment:
<p>I would use <code>beta_low = beta_eti.sel(quantile=0.025)</code>  both changing positional indexing to label based indexing and hpd to eti as I mentioned in the original comment in the issue. </p><p> </p><p>Note, it's also possible to use <code>beta_eti.isel(quantile=0)</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>